platform/android/display_server_android.cpp

```diff
@@ -593,6 +593,7 @@ DisplayServerAndroid::DisplayServerAndroid(const String &p_rendering_driver, Dis
 #if defined(GLES3_ENABLED)
 	if (rendering_driver == "opengl3") {
 		RasterizerGLES3::make_current(false);
+		OS::get_singleton()->set_gles_over_gl(false);
```

Comment:
I guess we can replace `RasterizerGLES3::is_gles_over_gl()` with `OS::get_singleton()->get_gles_over_gl()` and remove argument from `make_current` to avoid duplication.

Reply:
Well I meant more just doing this:
```diff
diff --git a/drivers/gles3/rasterizer_gles3.h b/drivers/gles3/rasterizer_gles3.h
index 80a4a792bb..e3558fc825 100644
--- a/drivers/gles3/rasterizer_gles3.h
+++ b/drivers/gles3/rasterizer_gles3.h
@@ -113,6 +113,7 @@ public:
 
 	static void make_current(bool p_gles_over_gl) {
 		gles_over_gl = p_gles_over_gl;
+		OS::get_singleton()->set_gles_over_gl(p_gles_over_gl);
 		_create_func = _create_current;
 		low_end = true;
 	}
```

Changing the GLES3 renderer code everywhere to check this in OS could be done, but I think there's value in this staying self-contained for GLES3.